src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => (b !== 0 ? a / b : 0),
```

Comment:
The `add`, `subtract`, `multiply`, and `divide` methods currently return the result of the operation without updating the `result` property of the calculator object. To align with the expected functionality, these methods should modify the `result` property directly.